indico_toolkit/pipelines/file_processing.py

```diff
@@ -45,6 +45,16 @@ def get_file_paths_from_dir(
             f"Found {len(self.file_paths)} valid files and {len(self.invalid_suffix_paths)} paths with invalid suffixes."
         )
 
+    def move_all_file_paths(
+        self, 
+        path_to_dir,
+        destination_dir,
+        accepted_types: Tuple[str] = ("pdf", "tiff", "tif", "doc", "docx")
```

Comment:
we should probably refactor this and line 21. Make `accepted_types` a member variable and reference as `self.accepted_types`